### PR TITLE
Added a build script

### PR DESCRIPTION
**Pull request by nraddy**
Moved the changes to gophpr package.

CC: @wpbrandon , @paulanunda , @rstuckless

### CODE REVIEW

**Comment by wpbrandon** [comment]
Should there be any tests added on the php container level to validate the existence or functionality of the extenstions?

**Comment by nraddy** [comment]
Good question, I will check with Brian on why tests are being added to gophpr, but it would make sense if these php dockers are going to be used across many different components. Right now, these are built for gophpr.

**Comment by jcshort** [comment]
We may want to do this in our intermediate layer rather than the public image

File: Dockerfile.php5.6 [hunk 1]
```diff
@@ -95,3 +97,12 @@ RUN set -ex; \
...
 		echo "extension=newrelic.so"; \
 	} > /usr/local/etc/php/conf.d/docker-php-ext-newrelic.ini
+
+RUN set -ex; \
+	\
+	{ \
+		echo '[runkit]'; \
+	} >> /usr/local/etc/php/php.ini; \
+	{ \
+		echo "extension=$(find / -path /tmp -prune -o -name 'runkit.so' -print)"; \
```
Comment (jcshort) [inline Dockerfile.php5.6]: i believe these are zend_extensions.  extension/zend_extension also do not require their own INI section headers.

Comment (nraddy) [inline Dockerfile.php5.6]: Tried without, did not work without this INI section.

**Comment by nraddy** [comment]
Moved all the changes to gophpr, left the build script addition here.

**Pull request merged**